runtime/src/main/java/com/dropbox/kaiken/runtime/InjectorFactory.kt

```diff
@@ -16,13 +16,13 @@ import com.dropbox.kaiken.Injector
  *
  * ```
  */
-interface InjectorFactory<InjectorType : Injector> {
+fun interface InjectorFactory<InjectorType : Injector> {
```

Comment:
backwards compatible change that simplifies call sites

Reply:
Beautiful